components/root/root.test.jsx

```diff
@@ -147,6 +172,25 @@ describe('components/Root', () => {
         wrapper.unmount();
     });
 
+    test('should reload on focus after getting signal login event from another tab', () => {
+        Object.defineProperty(window.location, 'reload', {
+            configurable: true,
+            writable: true,
+        });
+        window.location.reload = jest.fn();
+        const wrapper = shallow(<Root {...baseProps}/>);
+        const loginSignal = new StorageEvent('storage', {
+            key: StoragePrefixes.LOGIN,
+            newValue: String(Math.random()),
+            storageArea: localStorage,
+        });
+
+        window.dispatchEvent(loginSignal);
+        document.dispatchEvent(new Event('visibilitychange'));
+        expect(window.location.reload).toBeCalledTimes(1);
+        wrapper.unmount();
+    });
+
     describe('onConfigLoaded', () => {
         // Replace loadMeAndConfig with an action that never resolves so we can control exactly when onConfigLoaded is called
         const props = {
```

Comment:
Looks like this `loadMeAndConfig` on line 200 can also be removed, and this can just be `const props = baseProps;` or `const props = {...baseProps};`

Reply:
oh yeah it can be just baseProps, i removed the props

Reply:
✅  Updated as per request/discussion